docs/releases/2.0.txt

```diff
@@ -140,7 +140,8 @@ CSRF
 Database backends
 ~~~~~~~~~~~~~~~~~
 
-* ...
+* Oracle database introspection now works with auto fields, but only for newly
```

Comment:
I think I'd put this under "Management commands", with wording like "On Oracle, inspectdb can now introspect AutoField, if the column is created as an identity column."